inc/question.class.php

```diff
@@ -755,6 +739,14 @@ public function post_purgeItem() {
    }
 
    public function showForm($ID, $options = []) {
+      // $options['candel'] = false;
+      // $options['formoptions'] = sprintf('data-itemtype="%s"', self::getType());
+      // TemplateRenderer::getInstance()->display('@formcreator/pages/question.html.twig', [
+      //    'item' => $this,
+      //    'params' => $options,
+      // ]);
+      // return true;
```

Comment:
too much complex to convert now, lets keep this draft and continue in an other PR